src/condor_dagman/dagman_submit.cpp

```diff
@@ -814,7 +814,7 @@ direct_condor_submit(const Dagman &dm, Job* node,
 
 bool send_reschedule(const Dagman & dm)
 {
-	if (!dm.useDirectSubmit)
+	if (dm.options[deep::i::SubmitMethod] == 0)
```

Comment:
The thought did cross my mind. Especially since in `dag.cpp` I am using this in coordination with a switch statement for deciding which submission method to use.

Reply:
I've approved the PR; you can make the change on this PR or save it for later, as you prefer.

Reply:
I will defer this for part two of this work (using the options class deeper in DAGMan rather than copying everything down)